gui.py

```diff
@@ -1180,7 +1186,7 @@ def build_train_tab():
                             info=loc.localize("training-tab-classifier-textbox-info"),
                         )
                         output_format = gr.Radio(
-                            ["tflite", "raven", "both"],
+                            ["tflite", "raven", (loc.localize("training-tab-output-format-both"),"both")],
```

Comment:
The `both` afterwards is unessary, right?

Reply:
I assumed its passed as value, so we need the 'both' so we dont pass the translated value to the script.